testsuite/MDAnalysisTests/test_distances.py

```diff
@@ -63,12 +67,20 @@ def mindist(a, b, box):
             return np.linalg.norm(x - np.rint(x / box) * box)
 
         ref = mindist(a, b, box)
-        val = MDAnalysis.lib.distances.distance_array(np.array([a]), np.array([b]), box)[0, 0]
+        val = MDAnalysis.lib.distances.distance_array(np.array([a]), np.array([b]),
+                                                      box=box, mode=self.mode)[0, 0]
 
-        assert_almost_equal(val, ref, decimal=6, err_msg="Issue 151 not correct (PBC in distance array)")
+        assert_almost_equal(val, ref, decimal=6,
+                            err_msg="Issue 151 not correct (PBC in distance array)")
 
+class TestDistanceArray_Serial(_TestDistanceArray):
+    mode = "serial"
 
-class TestDistanceArrayDCD(TestCase):
+class TestDistanceArray_OpenMP(_TestDistanceArray):
```

Comment:
I'm not 100% sure that travis will a) build in openmp and b) run with more than one thread (and find race conditions)

But it's still good to have the tests obviously